modules/quickstart-opentelemetry.adoc

```diff
@@ -66,22 +68,17 @@ The `ClusterRole` resource is created automatically during the Cluster Logging O
 +
 [source,terminal]
 ----
-$ oc project openshift-logging
+$ oc adm policy add-cluster-role-to-user collect-application-logs -z collector -n openshift-logging
 ----
 +
 [source,terminal]
 ----
-$ oc adm policy add-cluster-role-to-user collect-application-logs -z collector
+$ oc adm policy add-cluster-role-to-user collect-audit-logs -z collector -n openshift-logging
 ----
 +
 [source,terminal]
 ----
-$ oc adm policy add-cluster-role-to-user collect-audit-logs -z collector
-----
-+
-[source,terminal]
-----
-$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z collector
+$ oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z collector -n openshift-logging
 ----
 +
```

Comment:
How about move  (Create a `UIPlugin` CR to enable the *Log* section in the *Observe* tab)  after line 45.   
The UIPlugin display logs from lokistack,  it doesn't care about if ClusterLogForwarder is running or not.